Improve bug reference

MikeBazMSFT opened the pull request:
There is a bug mentioned as "current" that is in fact resolved in a later version.  At some future time, we should remove the reference, but I feel that it was resolved too recently to remove all references to it at this time.

azurecla commented:
Hi __@janety__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (janety). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

azurecla commented:
Hi __@janety__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (janety). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Janet, can you please edit your profile per these parameters so we know you're a Microsoft employee (so we don't have to repeatedly look you up in the GAL, and for other reasons, too)?
https://github.com/Azure/azure-content-pr/blob/master/contributor-guide/tools-and-setup.md#create-a-github-account-and-set-up-your-profile
Thanks

janety commented:
Sure thing- didn’t realize there was a convention.  I’ve just made the updates.

Thanks
Janet

From: Ross McAllister [mailto:notifications@github.com]
Sent: Wednesday, November 18, 2015 2:34 PM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Janet Yeilding <Janet.Yeilding@microsoft.com>
Subject: Re: [azure-content] Update sql-database-develop-java-simple-windows.md (#4996)


Janet, can you please edit your profile per these parameters so we know you're a Microsoft employee (so we don't have to repeatedly look you up in the GAL, and for other reasons, too)?
https://github.com/Azure/azure-content-pr/blob/master/contributor-guide/tools-and-setup.md#create-a-github-account-and-set-up-your-profile
Thanks

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/4996#issuecomment-157886671>.

MikeBazMSFT committed:
Improve bug reference

There is a bug mentioned as "current" that is in fact resolved in a later version.  At some future time, we should remove the reference, but I feel that it was resolved too recently to remove all references to it at this time.
